form.yml.erb

```diff
@@ -7,6 +7,9 @@
 cluster:
   - "owens"
   - "pitzer"
+<% if !OodAppkit.clusters["kubernetes"].nil? && OodAppkit.clusters["kubernetes"].allow? -%>
+  - "kubernetes"
+<% end -%>
```

Comment:
I know you left a comment on jupyter, but we shouldn't need this. Also - I think they're symbols not strings, just FYI.

Reply:
I removed the ACL checks, and it was working with stings for `kubernetes` key though symbols may have also worked